src/components/App.js

```diff
@@ -4,6 +4,7 @@ import {
   Route,
```

Comment:
Why is the App component in the **component** folder? we agreed that all stateful components should be in the container folder. Secondly, your task is only about Travis, why are you editing other areas, please stay in your boundary.

Reply:
> Why is the App component in the **component** folder? we agreed that all stateful components should be in the container folder. Secondly, your task is only about Travis, why are you editing other areas, please stay in your boundary.

@Stanley-Okwii , I thought because **App**  component didn't initialize state and that it did not have any redux dispatch actions that it deserved to be in the components folder. Am fixing this now. Will let you know when it's done.